test/test_torch.py

```diff
@@ -3448,11 +3442,10 @@ def test_clone_all_dtypes_and_devices(self):
         for device in torch.testing.get_all_device_types():
             for dt in torch.testing.get_all_dtypes():
                 x = torch.tensor((1, 1), dtype=dt, device=device)
+                y = x.clone()
                 if (device == 'cuda' and dt == torch.bfloat16):
-                    self.assertRaises(RuntimeError, lambda: x.clone())
+                    self.assertRaises(RuntimeError, lambda: self.assertEqual(x, y))
```

Comment:
can you explain what's going on here?  I presume this assert was in to check that clone threw an exception with cuda+bfloat16, but now it's throwing an exception on the comparison?  That seems weird?

Reply:
This is funny one, as assert equals not yet functional for bfloat16, however because I fixed x.clone(), I had to update assertRaises piece.

Reply:
right, the first part makes sense because the assert is there so you have to update the test, so we are sure things get tested (otherwise these exceptional cases just live in the code forever and we never actually get test coverage).

But the second part doesn't make sense to me -- why does assertEqual throw?

Reply:
Because `x -y` throws =)

Reply:
ah, I think @izdeby has a fix for that in another PR (https://github.com/pytorch/pytorch/pull/22851).  There might be a conflict between those two, just a heads up.